src/pages/merge/bro/index.tsx

```diff
@@ -15,6 +17,16 @@ const BroMergePage = () => {
   const { data: broContractBalance } = useTokenBalance(CONTRACT_ADDRESSES.broEater, CONTRACT_ADDRESSES.wewe);
   const { data: bbroContractBalance } = useTokenBalance(CONTRACT_ADDRESSES.bbroEater, CONTRACT_ADDRESSES.wewe);
 
+  const getDaysRemaining = () => {
+    const today = new Date();
+    const startDate = new Date("2024-10-07 16:30:00");
+    const timeDiff = today.getTime() - startDate.getTime();
+    return Math.ceil(timeDiff / (1000 * 3600 * 24));
```

Comment:
day.js library already installed on the project. You can use it to calculate the difference: https://day.js.org/docs/en/display/difference

In this case it's not a big issue, but sometimes it can go wrong. For example, consider daylight saving changes